test/docfx.Test/lib/MarkdigUtilityTest.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Markdig;
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    public static class MarkdigUtilityTest
+    {
+        private static MarkdownPipeline _markdownPipeline = new MarkdownPipelineBuilder().UseYamlFrontMatter().Build();
+
+        [Theory]
+        [InlineData("abc", true)]
+        [InlineData("# ABC", true)]
+        [InlineData("[link](https://github.com)", true)]
+        [InlineData("![image](image.png)", true)]
+        [InlineData("<img src=\"imag.png\"/>", true)]
+        [InlineData("<img src=\"imag.png\">abc</img>", true)]
+        [InlineData("<a href=\"www.microsoft.com\"><img src=\"imag.png\"/></a>", true)]
+        [InlineData("<a>test</a>", true)]
+        [InlineData("`code`", true)]
+
+        [InlineData("<!--comments-->", false)]
+        [InlineData("    ", false)]
```

Comment:
Would you like to add a test case about "&nbsp;" and TripleColonBlock of image?

Reply:
ok, i will add